cuegen.py

```diff
@@ -65,7 +74,7 @@ def parse(self, line: str):
         line = re.sub(f"(\\[|\\(|){match_time.group(1)}(\\]|\\)|)", "", line)
         artist = None
         title = None
-        for split in line.split("-"):
+        for split in line.split(" - "):
```

Comment:
this, actually, is not a bug. it's made on purpose, because artist name or title often can have hyphen.

Reply:
That's why I thought it was a bug. For example if you have artist-name - track title it will use artist as the performer and name - track as the track name whereas it should be performer = artist-name track=title.

Unless whatever you are using to create the timed list is different. That's how Audacity would create a label track.

Reply:
Sorry, I just realized that there was a bug. You are right, it was a rough morning :smile:

Reply:
No problem. Glad I was able to help.